packaging/examples/kafka/kraft/README.md

```diff
@@ -0,0 +1,10 @@
+# KRaft examples
+
+The examples in this directory show how you can use Kraft (ZooKeeper-less Apache Kafka) with Strimzi.
+* The [`kafka.yaml`](kafka.yaml) deploys a Kafka cluster with one pool of _KRaft controller_ nodes and one pool of _KRaft broker_ nodes.
+* The [`kafka-ephemeral.yaml`](kafka-ephemeral.yaml) deploys a Kafka cluster with one pool of _KRaft controller_ nodes, one pool of _KRaft broker_ nodes and ephemeral storage.
+* The [`kafka-with-dual-role-nodes.yaml`](kafka-with-dual-role-nodes.yaml) deploys a Kafka cluster with one pool of KRaft nodes that share the _broker_ and _controller_ roles.
+
+If you want to use KRaft, you have to make sure that the `KafkaNodePools` and `UseKRaft` feature gates are not disabled.
+Please be aware that ZooKeeper-less Apache Kafka is still a work in progress and is still missing some important features.
+For example JBOD storage is not supported (you can use the `type: jbod` storage in the Strimzi custom resources, but it should contain only a single volume)
```

Comment:
```suggestion
To use KRaft, ensure that the `KafkaNodePools` and `UseKRaft` feature gates are enabled. 
Please note that ZooKeeper-less Apache Kafka is still under development and lacks some critical features. 
For instance, JBOD storage is not supported (you can specify `type: jbod` storage in Strimzi custom resources, but it should contain only a single volume).
```

Reply:
Your wording is confusing -> the feature gates are enabled by default. The way you phrase it, everyone will investigate if it is enabled because they did not enable it. But they will know that they did not disable it.

Reply:
(I applied the rest, just kept the part around it not being disabled)

Reply:
> (I applied the rest, just kept the part around it not being disabled)
Yeah, agree that part was quite confusing. When I read it again. Thanks